schema/artist/statuses.js

```diff
@@ -10,61 +10,72 @@ const ArtistStatusesType = new GraphQLObjectType({
   fields: {
     artworks: {
       type: GraphQLBoolean,
+      resolve: ({ published_artworks_count }) => !!published_artworks_count,
```

Comment:
what's it mean for `!!` on an integer? Should it just be `published_artworks_count > 0` (since it can never be nil)